requirements_pacman.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# Source: https://github.com/gfduszynski/cm-rgb/issues/45#issuecomment-920936950
+sudo pacman -S \
+  cairo \
+  gobject-introspection \
```

Comment:
This fails with 404

Reply:
I have tested with Manjaro 21.1.6 live boot environment and it as working. Is it still a problem for you?

Reply:
I have Manjaro `21.2.0` and for some reason the package didn't install.
I'll try again.

Reply:
I found my bug.
I had switched Manjaro branches from stable to unstable. Then back. And I didn't update the repository links.
Turns out that my scripts were working :tada: :+1:

Reply:
Confirmed. Works.